### PR TITLE
How to do a PR + some troubleshooting additions

### PR DESCRIPTION
**Pull request by jmatheson**
Based upon conversations and questions I've helped with in Gitter

### CODE REVIEW

File: docs/Resources/my-first-pr.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+2. Github will automatically take you to your copy (notice in the address bar you are now in your own personal github directory)
+3. In the branch drop down on the upper left of the page we are going to select the dev branch, since that is the one we want to edit. 
+4. Now we need to find the file we want to edit. Click through the directory structure until you find and are looking at the content of the file you want to change.
+5. Next, press the pencil icon in the upper right next to the trash can icon.
+6. Make edits to the file as necessary.
+7. Next we want to commit our changes. But first we should note what we changed and why. Be sure to put a one liner explaining the why of making the changes you did.
+8. Commit the changes.
+9. Now look and make sure everyting you changed looks like you meant it to (no typos, etc). If any problems, go back and edit again and save again.
+
+We now have an improved vile that we want to be pulled back into the dev repository at https://github.com/openaps/docs
```
Comment (scottleibrand) [inline docs/Resources/my-first-pr.md]: s/vile/file/

File: docs/Resources/my-first-pr.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Go to https://github.com/openaps/docs
+2. Select dev from the branch drop down on the upper left (since that's where we'd like the changes to go to)
+3. Press the pull request button
+4. Set up the the branches you are targeting. The easiest way of thinking about the branch range is this: the base branch is where you think changes should be applied, the head branch is what you would like to be applied.
+5. So choose so the base fork as openaps/docs and then the base as dev. The head fork is going to be <youraccount>/docs and the base as dev. 
+6. It should show the list of changes you made. If not, you did something wrong so stop here and ask for help. If the list looks like your changes then  put a note in there to what the overarching reason for the changes are (in your case you only made one, but you could have made a bunch). Create the PR.
```
Comment (scottleibrand) [inline docs/Resources/my-first-pr.md]: two spaces between "then" and "put"

File: docs/Resources/troubleshooting.md [hunk 3]
```diff
@@ -57,3 +57,20 @@ To fix a corrupted git repository you can run `oref0-fix-git-corruption.sh`, it
 ### Environment variables
 
 If you are getting your BG from Nightscout or you want to upload loop status/resuts to Nightscout, among other things you'll need to set 2 environment variabled: `NIGHTSCOUT_HOST` and `API_SECRET`. If you do not set and export these variables you will receive errors while running `openaps report invoke monitor/ns-glucose.json` and while executing `ns-upload.sh` script which is most probably part of your `upload-recent-treatments` alias.Make sure your `API_SECRET` is in hashed format. Please see [this page](https://github.com/openaps/oref0#ns-upload-entries) for details. Additionally, your `NIGHTSCOUT_HOST` should be in a format like `http://yourname.herokuapp.com` (without trailing slash). For the complete visualization guide use [this page](https://github.com/openaps/docs/blob/master/docs/Automate-system/vizualization.md) from the OopenAPS documentation.
+
+### Common error messages
+
+#### Don't have permission, permission not allowed, etc
+
+The command you are running likely needs to be run with root permissions, try the same command again with ```sudo ``` in front of it
```
Comment (scottleibrand) [inline docs/Resources/troubleshooting.md]: Should add a warning not to do any `sudo openaps` commands, or your .git permissions will get messed up.

**Comment by scottleibrand** [comment]
This looks good.  You want to fix the nits I commented above before I merge, or shall I merge and fix those in dev?

**Comment by jmatheson** [comment]
Either way is fine. Agree with the suggested changes. I will check tonight and if you have not merged i can add them.

**Comment by danamlewis** [comment]
Second +1, ready to merge once those tweaks are made, @jmatheson

**Comment by jmatheson** [comment]
@scottleibrand @danamlewis i think we are ready to go

**Pull request merged**